models.py

```diff
@@ -30,13 +36,15 @@ def get_username(self):
 class favorite_restraunts(UserMixin, db.Model):
     id = db.Column(db.Integer, primary_key=True)
     RestaurantName = db.Column(db.String(120))
-    user_id = db.Column(db.Integer, db.ForeignKey("user.id"))
+    yelpRestrauntID = db.Column(db.String(120))
+    user_id = db.Column(db.Integer, db.ForeignKey("user.id", ondelete="CASCADE"))
 
 
 class friends(UserMixin, db.Model):
```

Comment:
would it also be possible to store the friend profile pic here? 

The way the profile page is set up right now, each friend in the friend list gets their profile picture from searching through the friends table using the friendID as the userID, and then the profile picture is combined in an object with all the information from the friends result and passed into the front end. Since all that functionality is already in place there may not be that much benefit at this point to adding it, but it can also give us cleaner and faster code since we don't need to go through two tables each time.

Not necessary, just a consideration